assets/scss/components/foundation/modal/_modal.scss

```diff
@@ -1,6 +1,7 @@
 // =============================================================================
 // MODAL (CSS)
 // =============================================================================
+// Fix for content shifted to top in modal window when bottom variant option selected
```

Comment:
Could we add comment above the fix but not at the top of file?

Reply:
Yes we can. I add the comment in such way trying to inherit style of css comments we already have? So should we?
<img width="780" alt="Screenshot 2020-10-01 at 10 40 51" src="https://user-images.githubusercontent.com/66325265/94781717-e2526c00-03d2-11eb-982d-4a70cba60512.png">